<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gameView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QComboBox" name="locationColection">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showMapButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmActionButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Action</string>
    </property>
   </widget>
   <widget class="QComboBox" name="nextActionCollection">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>300</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="endTurnButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>End turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardBagButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>370</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Bag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendAgentButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Agent</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="playerStatusTextBrowser">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
